### PR TITLE
Updates to minimal and minimal-sops examples

### PR DESCRIPTION
**Pull request by ivanfetch**
* Updates to the `minimal` example:
	* Update the Dockerfile to copy the app, avoiding populating the rest of this example in the Docker image.
	* Updates to the ReadMe to clarify specifying your image repository, linking to other files in the repo as references, add a clean-up section, and other clarity / typo adjustments.
	* Clarify what should be replaced in the Kubernetes deployment image specification.
* Replace the sops example with one based on the minimal example:
	* The app outputs the environment variable that maps to the secret in Kubernetes, to show that the secret was deployed by rok8s-scripts.
	* Document encrypting a secret using a KMS key in AWS or GCP.

### CODE REVIEW

**Comment by ivanfetch** [comment]
This PR is ready for review. I've tested the `minimal-sops-secrets` example in a repo in my own Github account.

File: docs/secrets.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Managing Kubernetes Secrets Securely
 There are multiple ways to securely manage Kubernetes Secrets with rok8s-scripts. With support for AWS Secrets Manager, Sops encrypted secrets, and fetching secrets from external sources like AWS S3 or Google Cloud Storage, there should be at least one option that works well for you.
 
+The Rok8s-scripts [k8s-deploy-secrets helper script](./../bin/k8s-deploy-secrets) decrypts and deploys secrets to Kubernetes. You do not need to call this script directly, it is called by other `deploy` scripts.
```
[the remark this answers is not in the document]
Comment (ivanfetch) [reply docs/secrets.md]: I made paths absolute that are not referencing something within the same directory - so the path to the script in `bin` and to another doc in `docs` are now absolute. I think the `./file` or `./sub-dir/file` relative references are ok; not confusing.

File: examples/minimal-sops-secrets/README.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+we run some of the scripts provided by rok8s-scripts. In particular, we use:
+
+* `docker-pull` to get the last image we pushed, which warms the local docker cache for faster builds
+* `docker-build` to build the image for the current commit
+* `docker-push` to push the image for the current commit to our image repository
+* `prepare-kubectl` to configure the `kubectl` command to be able to deploy resources to our Kubernetes cluster
+* `k8s-deploy-and-verify` to deploy our image to Kubernetes and make sure the deployment succeeded
+	* This also calls the `k8s-deploy-secrets` script to decrypt and deploy secrets
+
+We also use the rok8s-scripts CI image, `quay.io/reactiveops/ci-images:v9-stretch`,
```
[the remark this answers is not in the document]
Comment (ivanfetch) [reply examples/minimal-sops-secrets/README.md]: Definitely. This was already the case in CI config - ReadMes now updated.

File: examples/minimal-sops-secrets/.circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  deploy: &deploy
+    run:
+      name: Deploy to Production
+      command: |
+        docker-push -f deploy/production.config
+        prepare-kubectl
+        k8s-deploy-and-verify -f deploy/production.config
+
+jobs:
+  build_image:
```
[the remark this answers is not in the document]
Comment (ivanfetch) [reply examples/minimal-sops-secrets/.circleci/config.yml]: True - I varied the name of the task to help clarify.

File: examples/minimal-sops-secrets/README.md [hunk 4]
```diff
@@ -107,7 +107,7 @@ Commit and push your changes to the master branch to trigger another build.
 Everything should run successfully this time, and you should see your app
 running inside your Kubernetes cluster.
 
-You can check your app by running
+You can check your app by runningz
```
Comment (sudermanjr) [inline examples/minimal-sops-secrets/README.md]: Extra `z` here I think

**Pull request merged**